test/k8sT/manifests/demo-graceful-term.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: testclient
```

Comment:
Don't we have multiple similar pods already available?

Reply:
Nope, they are replicated in every e2e test that need the demo pods. Anyway I'm going to fold this into the graceful-test yaml file.